src/main/kotlin/view/FilterPanel.kt

```diff
@@ -41,7 +41,7 @@ class FilterPanel : View() {
                     addClass(CssStyle.checkBox)
                     checkbox(s) {
                         action {
-                            imageController.applyFilter(op)
+                            imageController.applyFilter(op, isSelected)
```

Comment:
Again, we don't need `isSelected`. Comparing raw and result image can happen only inside/involving the controller. No need to add a checkbox to toggle